components/inputs/test/input-checkbox.visual-diff.html

```diff
@@ -32,17 +32,27 @@
 			<d2l-input-checkbox id="wc-disabled-indeterminate" indeterminate disabled>Checkbox (indeterminate, disabled)</d2l-input-checkbox>
 		</div>
 		<div class="visual-diff">
-			<d2l-input-checkbox id="wc-multiline" style="width: 200px;">
+			<d2l-input-checkbox id="wc-multiline" style="overflow: hidden; width: 200px;">
```

Comment:
Added these `overflow: hidden;` to make it more obvious if there is a regression. Without this, it's not particular obvious when there is a problem because our visual diff captures a slightly larger area.